PriceCalculatorsKataTests/ProductTest.cs

```diff
@@ -0,0 +1,55 @@
+using NUnit.Framework;
+using PriceCalculatorKata;
+namespace PriceCalculatorsKataTests
+{
+    public class Tests
+    {
+        [SetUp]
+        public void Setup()
+        {
+        }
+
+        [Test]
+        public void productPricePercision()
+        {
+            var roundedUpPriceProduct = new Product(0)
+            {
+                BasePrice = 20.256f,
+                Name = "testing product",
+                TaxPercentage = 20,
+            };
+            var roundedDownPriceProduct = new Product(1)
+            {
+                BasePrice = 20.254f,
+                Name = "testing product",
+                TaxPercentage = 20,
+            };
+
+            Assert.AreEqual(20.26f, roundedUpPriceProduct.BasePrice);
```

Comment:
nit: maybe it's better to have separated tests to cover those cases, and not putting them into one case ? let's talk about it.